app/helpers/provider_helper.rb

```diff
@@ -1,14 +1,16 @@
 module ProviderHelper
-  def add_course_url(email, provider)
+  def add_course_url(email, provider, current)
+    cycle_key = current ? 'current_cycle' : 'next_cycle'
+
     if provider.accredited_body?
-      google_form_url_for(Settings.google_forms.new_course_for_accredited_bodies, email, provider)
+      google_form_url_for(Settings.google_forms[cycle_key].new_course_for_accredited_bodies, email, provider)
     else
-      google_form_url_for(Settings.google_forms.new_course_for_unaccredited_bodies, email, provider)
+      google_form_url_for(Settings.google_forms[cycle_key].new_course_for_unaccredited_bodies, email, provider)
     end
   end
 
-  def add_course_link(email, provider)
-    link_to "Add a new course", add_course_url(email, provider), class: "govuk-button govuk-!-margin-bottom-2", rel: "noopener noreferrer", target: "_blank"
+  def add_course_link(email, provider, current)
```

Comment:
You might make this method signature clearer by:
- using a named parameter
- expanding the variable name (`current` is a bit ambiguous)
- naming it in a way that makes it clear that it's a boolean

eg.
```suggestion
  def add_course_link(email, provider, is_current_cycle:)
```

Reply:
Added in c4ef56d5553c66e8c58788f5bfe2437f546cc35a